src/Container.php

```diff
@@ -178,7 +178,8 @@ public static function create(): self
             },
             SourceFileFilter::class => static function (self $container): SourceFileFilter {
                 return new SourceFileFilter(
-                    $container->getConfiguration()->getSourceFilesFilter()
+                    $container->getConfiguration()->getSourceFilesFilter(),
+                    $container->getSchemaConfiguration()->getSource()->getExcludes()
```

Comment:
That doesn't look right to me: either we should move it up in `Configuration` or find something else. The schema configuration is meant to be a 1:1 dependency with the configuration file, which is a "low level" thing: it is not something we want the "domain" of Infection to depend on

Reply:
See https://github.com/infection/infection/pull/1241